uPortal-webapp/src/main/resources/properties/jgroups.xml

```diff
@@ -27,66 +27,58 @@
 -->
 <config xmlns="urn:org:jgroups"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="urn:org:jgroups http://www.jgroups.org/schema/JGroups-3.1.xsd">
-    <TCP bind_port="${uPortal.cacheManager.jgroups.tcp.bind_port}"
-         loopback="false"
-         recv_buf_size="${tcp.recv_buf_size:20M}"
-         send_buf_size="${tcp.send_buf_size:640K}"
+        xsi:schemaLocation="urn:org:jgroups http://www.jgroups.org/schema/JGroups-3.6.xsd">
```

Comment:
:question: are jGroups changes indended to be in this PR?

Reply:
Nope